### PR TITLE
Update Readme with console link.

### LINKED ISSUE #18: argocd access
As an Application Owner/Project-specific DevOps,
I want to get access to ArgoCD Web UI
so that I can see the status of my deployments

### PR DESCRIPTION
**Pull request by HumairAK**
Fixes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
-# opf-argocd-apps
-A repository to store Operate-First ArgoCD Applications
+# Operate-first ArgoCD application manifests
+A repository to store Operate-First ArgoCD Applications.  This repository's primarily purpose is to enable the [App of Apps Pattern](https://argoproj.github.io/argo-cd/operator-manual/cluster-bootstrapping/#app-of-apps-pattern). This means that we can create an ArgoCD application that can be pointed to this repo and deploy various other ArgoCD applications.
```
Comment (hemajv) [inline README.md:2]: change `primarily` to `primary`?

**Review by 4n4nd — APPROVED** [review]
:ship:

File: README.md (hunk 1 above)
Comment (HumairAK) [reply README.md]: updated

**Review by hemajv — APPROVED** [review]
:ship:

**Pull request merged**